README.md

```diff
@@ -64,175 +64,33 @@ Note: **Enable Bot Account Creation** must be enabled under System Console
 
 For use all API(V4.0.0), you need add bot user to system admin group to avoid 403 error.
```

Comment:
Is this still valid? It's from the legacy README.md but I don't think this is the case anymore?

Reply:
The description is a bit cryptic, but the bot does indeed need system admin permissions to use the entire API, e.g. ephemeral replies.

Reply:
Cool, I've updated the doc to clarify this a little better.